w3cdatalayer/application/factory.go

```diff
@@ -54,6 +59,7 @@ func (s *Factory) Inject(
 	},
 ) {
 	s.router = router2
+	s.logger = logger
```

Comment:
Could you please add details about to module to the logger?
`logger.WithField(flamingo.LogKeyModule, "w3cdatalayer")`

Reply:
Good point. Added it like you said.